chapters/functions.tex

```diff
@@ -1883,7 +1883,7 @@ \section{Function Inlining and Event Generation}\label{function-inlining-and-eve
 Has only an effect within a function declaration
 
 If \lstinline!GenerateEvents = true!, the model developer proposes that crossing functions in the function should generate events (one possibility of doing this is to inline the function and generate events for the inlined function).
-
+As defaults this implies \lstinline!Inline = true!, but it can be overriden by specifying one of the inline-annotations.
```

Comment:
I don't think this is ready for a proper review, but I can already note that this doesn't rule out combining `GenerateEvents = true` with `Inline = false`, which I thought was generally perceived as an undesired combination.

Reply:
I remember asking this question in the meeting but now I don't remember the answer: What happens if GenerateEvents = true and the function cannot be inlined?

Reply:
This is precisely the reason why `GenerateEvents = true` should imply inlining, meaning that it is a tool failure if it can't be inlined.  Put differently, the point is that tools are not expected to be able to handle inlining by any other means than by inlining.

Reply:
This conversation is obsolete in view of later conversations and current state of the changes in this PR.  Resolving.